doc/jobs/miscellaneous_jobs.rst

```diff
@@ -51,3 +55,18 @@ repositories:
 
   Each page shows the targets defined by all the release build files using
   the same *target* repository.
+
+* The ``release-compare-page`` job, generated on the farm by
+  **generate_release_compare_page_job.py**,
+  invokes the script
+  *build_release_compare_page.py*.
+  The generated page shows, for a specific pair of ROS distributions, how the versions of the
+  repositories released in the earlier distribution compare in the more recent distribution.
+
+* The ``blocked-releases-page`` job, generated on the farm by
+  **generate_blocked_releases_page_job.py**,
+  invokes the script
+  *build_blocked_releases_page.py*.
+  The generated page shows, for a specific ROS distribution, which repositories from the previous 
+  distribution have not yet been released, and which repositories they are preventing from being 
+  released as a result.
```

Comment:
I think it might just apply to the second paragraph https://github.com/ros-infrastructure/ros_buildfarm/blob/master/scripts/generate_all_jobs.py#L116
I didn't have a particular reason for not including the blocked-releases-page job in the generate-all script, I just didn't see it. add it?